docs/howto/object-storage/s3/object-lock.md

```diff
@@ -0,0 +1,361 @@
+# Object Lock
```

Comment:
To be in line with the other page headings, "Object versioning" and "Object expiry", please lowercase the word "lock":

```suggestion
# Object lock
```

Do the same for other mentions of that term too, please.